### PR TITLE
feat: Add more optimization options

### PR DESCRIPTION
**Pull request by Walker-00**
Well I just added additional optimizations options for file size.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -3,6 +3,8 @@ resolver = "2"
 members  = [ "yazi-*" ]
 
 [profile.release]
+codegen-units = 1
+opt-level = 3
```
Comment (sxyazi) [inline Cargo.toml:7]: Seems `opt-level = 3` is the default value of releasing, https://doc.rust-lang.org/cargo/reference/profiles.html#release

Comment (Walker-00) [reply Cargo.toml:7]: ahh yea.

Comment (sxyazi) [reply Cargo.toml:7]: Okay, I'll only keep `codegen-units`.

**Comment by sxyazi** [comment]
Thank you!

**Pull request merged**